test/files/pos/t2377b/a.scala

```diff
@@ -0,0 +1,3 @@
+object Test {
+  (new Q.Inner).foo // make sure foo's type is well-formed
+}
```

Comment:
Both `typedIdent` and `typedSelect` were modified. Are both new code paths exercised by the test?